### PR TITLE
Astro 1.0.0

### PR DESCRIPTION
**Pull request by natemoo-re**
## Changes

- Releases Astro 1.0.0
- I already ran `changeset version`, so merging this will trigger the release directly! No additional PR.
- Packages **not** updated to 1.0.0
  - [x] `@astrojs/image`
  - [x] `@astrojs/mdx`
  - [x] `@astrojs/prefetch`

## Testing

N/A

## Docs

N/A

### CODE REVIEW

File: packages/astro/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
 # astro
 
+## 1.0.0
```
Comment (natemoo-re) [inline packages/astro/CHANGELOG.md:3]: Tried to put together a nice CHANGELOG using parts of the Migration Guide!

File: packages/integrations/netlify/CHANGELOG.md [hunk 2]
```diff
@@ -34,7 +47,6 @@
   The new `Astro.clientAddress` property allows you to get the IP address of the requested user.
 
   ```astro
-  <div>Your address {Astro.clientAddress}</div>
```
Comment (Princesseuh) [inline packages/integrations/netlify/CHANGELOG.md:37]: What happened here?

File: packages/create-astro/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # create-astro
 
+## 1.0.0
+
+### Major Changes
+
+- [`04ad44563`](https://github.com/withastro/astro/commit/04ad445632c67bdd60c1704e1e0dcbcaa27b9308) - > Astro v1.0 is out! Read the [official announcement post](https://astro.build/blog/astro-1/).
+
+  **No breaking changes**. This package is now officially stable and compatible with `astro@1.0.0`!
+
```
Comment (natemoo-re) [inline packages/create-astro/CHANGELOG.md:10]: This is the CHANGELOG for every package that isn't `astro`!

File: package.json [hunk 4]
```diff
@@ -65,15 +65,13 @@
       ]
     },
     "patchedDependencies": {
-      "@changesets/cli@2.23.0": "patches/@changesets__cli@2.23.0.patch",
-      "@changesets/assemble-release-plan@5.1.3": "patches/@changesets__assemble-release-plan@5.1.3.patch"
```
Comment (natemoo-re) [inline package.json:69]: Removed our custom `astro` logic!

File: packages/integrations/netlify/CHANGELOG.md (hunk 2 above)
Comment (natemoo-re) [reply packages/integrations/netlify/CHANGELOG.md:37]: Uh oh, I don't know! Prettier?

File: package.json [hunk 5]
```diff
@@ -65,15 +65,13 @@
       ]
```
Comment (matthewp) [inline package.json:65]: Shouldn't this file have `"version": "1.0.0"` like the other package.jsons have?

Comment (matthewp) [reply package.json:65]: Oh oops, this is the monorepo package.json, nevermind.

File: packages/astro/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "astro",
-  "version": "1.0.0-rc.8",
+  "version": "1.0.0",
```
Comment (matthewp) [inline packages/astro/package.json:3]: 🤯

File: packages/integrations/deno/CHANGELOG.md [hunk 7]
```diff
@@ -32,7 +40,7 @@
   The new `Astro.clientAddress` property allows you to get the IP address of the requested user.
 
   ```astro
-  <div>Your address {Astro.clientAddress}</div>
```
Comment (matthewp) [inline packages/integrations/deno/CHANGELOG.md:35]: Was this line removed on purpose?

File: packages/astro/CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,37 @@
...
+
+> Astro v1.0 is out! Read the [official announcement post](https://astro.build/blog/astro-1/).
+
+> **Note**: If you need help migrating an existing Astro project to the new Astro v1.0, check out our updated [Migration Guide](https://docs.astro.build/en/migrate/) and [full documentation website](https://docs.astro.build/).
```
Comment (sarah11918) [inline packages/astro/CHANGELOG.md:7]:
```suggestion
> **Note**
> If you need help migrating an existing Astro project to the new Astro v1.0, check out our updated [Migration Guide](https://docs.astro.build/en/migrate/) and [full documentation website](https://docs.astro.build/).
```
I think you should be able to do this for a formatted "GitHub Note" now, if you want!

File: packages/astro/CHANGELOG.md [hunk 9]
```diff
@@ -415,18 +447,10 @@ The **Astro v1.0.0 Release Candidate** comes includes new features, tons of bug
 
   Astro supports streaming in its templates. Any time Astro encounters an async boundary it will stream out HTML that occurs before it. For example:
 
-  ```astro
-
-  ```
-
```
Comment (natemoo-re) [inline packages/astro/CHANGELOG.md:421]: Erm... prettier removed the content here so I just removed the block entirely? Can probably dig up the old content, though.

File: packages/astro/CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,37 @@
...
+- [MDX Support](https://docs.astro.build/en/guides/integrations-guide/mdx/): A standard syntax for mixing UI components in Markdown is available with [`@astrojs/mdx`](https://docs.astro.build/en/guides/integrations-guide/mdx/).
+- [Vite 3.0](https://vitejs.dev/blog/announcing-vite3.html): An upgrade to our internal build engine, Vite.
+
+### Breaking Changes
+
+- `Astro.canonicalURL` has been deprecated. Please use [`Astro.url`](https://docs.astro.build/en/reference/api-reference/#astrourl) to construct your own canonical URL.
+- [CSS Specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) is now preserved for scoped styles. See the [migration guide](https://docs.astro.build/en/migrate/#changed-scoped-css-specificity) for more details.
+  > **Warning**: Please visually inspect your site output to make sure everything is styled as expected. If not, find your scoped style and increase the selector specificity manually to match the old behavior.
```
Comment (sarah11918) [inline packages/astro/CHANGELOG.md:22]:
```suggestion
  > **Warning**
  > Please visually inspect your site output to make sure everything is styled as expected. If not, find your scoped style and increase the selector specificity manually to match the old behavior.
```
Similarly, you should be able to make a fancy GitHub warning with this syntax.

File: packages/astro/CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,37 @@
...
+- Components and JSX in Markdown has been deprecated. For long-term support you should migrate to the [`@astrojs/mdx`](https://docs.astro.build/en/guides/integrations-guide/mdx/) integration. See the [migration guide](https://docs.astro.build/en/migrate/#deprecated-components-and-jsx-in-markdown) for more details.
+- The previously deprecated `<Markdown>` component has been removed. It is now available as a separate package, [`@astrojs/markdown-component`](https://github.com/withastro/astro/tree/main/packages/markdown/component).
+
+> **Note**: If you are coming from v0.25 or earlier, make sure you have read and followed the [v0.26 Migration Guide](https://docs.astro.build/en/migrate/#migrate-to-v026), which contained several major breaking changes.
```
Comment (sarah11918) [inline packages/astro/CHANGELOG.md:26]:
```suggestion
> **Note**
> If you are coming from v0.25 or earlier, make sure you have read and followed the [v0.26 Migration Guide](https://docs.astro.build/en/migrate/#migrate-to-v026), which contained several major breaking changes.
```
GitHub note syntax

File: packages/astro/CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,37 @@
...
+
+### Breaking Changes
+
+- `Astro.canonicalURL` has been deprecated. Please use [`Astro.url`](https://docs.astro.build/en/reference/api-reference/#astrourl) to construct your own canonical URL.
+- [CSS Specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) is now preserved for scoped styles. See the [migration guide](https://docs.astro.build/en/migrate/#changed-scoped-css-specificity) for more details.
+  > **Warning**: Please visually inspect your site output to make sure everything is styled as expected. If not, find your scoped style and increase the selector specificity manually to match the old behavior.
+- Components and JSX in Markdown has been deprecated. For long-term support you should migrate to the [`@astrojs/mdx`](https://docs.astro.build/en/guides/integrations-guide/mdx/) integration. See the [migration guide](https://docs.astro.build/en/migrate/#deprecated-components-and-jsx-in-markdown) for more details.
+- The previously deprecated `<Markdown>` component has been removed. It is now available as a separate package, [`@astrojs/markdown-component`](https://github.com/withastro/astro/tree/main/packages/markdown/component).
```
Comment (sarah11918) [inline packages/astro/CHANGELOG.md:24]: I'm not sure why these are looking in my preview as if they might be still part of the above warning? (The text is light grey in the file diff view on GitHub) So I guess just double-check any formatting here to make sure the list is rendering properly.

File: packages/integrations/netlify/CHANGELOG.md (hunk 2 above)
Comment (natemoo-re) [reply packages/integrations/netlify/CHANGELOG.md]: Looks like the same bug we hit here https://github.com/withastro/astro/pull/4159/files#diff-e3d0a49bea0efda551d3bd7e5e860ab65409efbb9a419d274b5806ccacdd4061

**Review by sarah11918 — COMMENTED** [review]
No typos!  🥳 

Just comments re: GitHub's newer "note" and "warning" syntax that will format them nicer than a standard blockquote: I think you need to have the title/type of aside be its own `>` then start the message on the next line with another `>`

File: packages/integrations/deno/CHANGELOG.md (hunk 7 above)
Comment (natemoo-re) [reply packages/integrations/deno/CHANGELOG.md:35]: Nope, see convo happening here https://github.com/withastro/astro/pull/4214#discussion_r941460217

File: packages/astro/CHANGELOG.md (hunk 9 above)
Comment (FredKSchott) [reply packages/astro/CHANGELOG.md:421]: ack, that's odd. cc @Princesseuh to take a look in case this is happening to users.

**Review by FredKSchott — APPROVED** [review]

File: packages/astro/CHANGELOG.md (hunk 12 above)
Comment (natemoo-re) [reply packages/astro/CHANGELOG.md:26]: You can view it as the final output here! https://github.com/withastro/astro/blob/66453ec6592612cf2b78b525e6db3c62bb7587b3/packages/astro/CHANGELOG.md

File: packages/astro/CHANGELOG.md (hunk 9 above)
Comment (Princesseuh) [reply packages/astro/CHANGELOG.md:421]: Taking a look, but I honestly have no idea what happened. The plugin ran fine for multiple formats until https://github.com/withastro/astro/pull/4159/files#diff-e3d0a49bea0efda551d3bd7e5e860ab65409efbb9a419d274b5806ccacdd4061 where it broke unexplainably 🤔

File: packages/integrations/netlify/CHANGELOG.md (hunk 2 above)
Comment (natemoo-re) [reply packages/integrations/netlify/CHANGELOG.md]: Manually fixed these for now. We'll have to investigate more!

**Pull request merged**